src/main/java/mate/academy/spring/validation/Password.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import mate.academy.spring.validation.impl.PasswordValidator;
+
+@Constraint(validatedBy = PasswordValidator.class)
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Password {
```

Comment:
```suggestion
public @interface MatchingPasswords {
```
и не увидела, чтобы ты где-то ее использовал